app/models/time_since.rb

```diff
@@ -0,0 +1,13 @@
+class TimeSince
```

Comment:
good idea to formalise this calculation  👍 👍   - but maybe this should be a service rather than a model ?